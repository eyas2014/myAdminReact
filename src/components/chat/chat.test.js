import React from 'react';
import Chat from './chat';
import enzyme from 'enzyme';

const wrapper=enzyme.shallow(<Chat></Chat>);

it ('should match the snapshot', ()=>{
	expect(wrapper).toMatchSnapshot();
})
