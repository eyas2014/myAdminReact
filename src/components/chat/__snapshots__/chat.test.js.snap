// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Chat />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header
          className="box-border-bottom"
        >
          <div
            className="btn"
          >
            <i
              className="far fa-comments"
            />
            <span
              className=""
            >
               Chat
            </span>
          </div>
          <div
            className="btn-group tools float-right"
            data-original-title="Status"
            data-toggle="tooltip"
            title=""
          >
            <button
              className="btn btn-default btn-sm active"
              type="button"
            >
              <i
                className="fa fa-square text-green"
              />
            </button>
            <button
              className="btn btn-default btn-sm"
              type="button"
            >
              <i
                className="fa fa-square text-red"
              />
            </button>
          </div>
        </header>,
        <main>
          <Conversations />
        </main>,
        <footer
          className="box-border-top"
        >
          <div
            className="input-group input-group-sm"
          >
            <input
              className="form-control"
              placeholder="Type message..."
            />
            <button
              className="input-group-addon"
              type="button"
            >
              <i
                className="fa fa-plus"
              />
            </button>
          </div>
        </footer>,
      ],
      "className": "chat",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="btn"
            >
              <i
                className="far fa-comments"
              />
              <span
                className=""
              >
                 Chat
              </span>
            </div>,
            <div
              className="btn-group tools float-right"
              data-original-title="Status"
              data-toggle="tooltip"
              title=""
            >
              <button
                className="btn btn-default btn-sm active"
                type="button"
              >
                <i
                  className="fa fa-square text-green"
                />
              </button>
              <button
                className="btn btn-default btn-sm"
                type="button"
              >
                <i
                  className="fa fa-square text-red"
                />
              </button>
            </div>,
          ],
          "className": "box-border-bottom",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <i
                  className="far fa-comments"
                />,
                <span
                  className=""
                >
                   Chat
                </span>,
              ],
              "className": "btn",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "far fa-comments",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " Chat",
                  "className": "",
                },
                "ref": null,
                "rendered": " Chat",
                "type": "span",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <button
                  className="btn btn-default btn-sm active"
                  type="button"
                >
                  <i
                    className="fa fa-square text-green"
                  />
                </button>,
                <button
                  className="btn btn-default btn-sm"
                  type="button"
                >
                  <i
                    className="fa fa-square text-red"
                  />
                </button>,
              ],
              "className": "btn-group tools float-right",
              "data-original-title": "Status",
              "data-toggle": "tooltip",
              "title": "",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fa fa-square text-green"
                  />,
                  "className": "btn btn-default btn-sm active",
                  "type": "button",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fa fa-square text-green",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "button",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fa fa-square text-red"
                  />,
                  "className": "btn btn-default btn-sm",
                  "type": "button",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fa fa-square text-red",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "button",
              },
            ],
            "type": "div",
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Conversations />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "data": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "main",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="input-group input-group-sm"
          >
            <input
              className="form-control"
              placeholder="Type message..."
            />
            <button
              className="input-group-addon"
              type="button"
            >
              <i
                className="fa fa-plus"
              />
            </button>
          </div>,
          "className": "box-border-top",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <input
                className="form-control"
                placeholder="Type message..."
              />,
              <button
                className="input-group-addon"
                type="button"
              >
                <i
                  className="fa fa-plus"
                />
              </button>,
            ],
            "className": "input-group input-group-sm",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "form-control",
                "placeholder": "Type message...",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fa fa-plus"
                />,
                "className": "input-group-addon",
                "type": "button",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fa fa-plus",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "button",
            },
          ],
          "type": "div",
        },
        "type": "footer",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header
            className="box-border-bottom"
          >
            <div
              className="btn"
            >
              <i
                className="far fa-comments"
              />
              <span
                className=""
              >
                 Chat
              </span>
            </div>
            <div
              className="btn-group tools float-right"
              data-original-title="Status"
              data-toggle="tooltip"
              title=""
            >
              <button
                className="btn btn-default btn-sm active"
                type="button"
              >
                <i
                  className="fa fa-square text-green"
                />
              </button>
              <button
                className="btn btn-default btn-sm"
                type="button"
              >
                <i
                  className="fa fa-square text-red"
                />
              </button>
            </div>
          </header>,
          <main>
            <Conversations />
          </main>,
          <footer
            className="box-border-top"
          >
            <div
              className="input-group input-group-sm"
            >
              <input
                className="form-control"
                placeholder="Type message..."
              />
              <button
                className="input-group-addon"
                type="button"
              >
                <i
                  className="fa fa-plus"
                />
              </button>
            </div>
          </footer>,
        ],
        "className": "chat",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="btn"
              >
                <i
                  className="far fa-comments"
                />
                <span
                  className=""
                >
                   Chat
                </span>
              </div>,
              <div
                className="btn-group tools float-right"
                data-original-title="Status"
                data-toggle="tooltip"
                title=""
              >
                <button
                  className="btn btn-default btn-sm active"
                  type="button"
                >
                  <i
                    className="fa fa-square text-green"
                  />
                </button>
                <button
                  className="btn btn-default btn-sm"
                  type="button"
                >
                  <i
                    className="fa fa-square text-red"
                  />
                </button>
              </div>,
            ],
            "className": "box-border-bottom",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <i
                    className="far fa-comments"
                  />,
                  <span
                    className=""
                  >
                     Chat
                  </span>,
                ],
                "className": "btn",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "far fa-comments",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " Chat",
                    "className": "",
                  },
                  "ref": null,
                  "rendered": " Chat",
                  "type": "span",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <button
                    className="btn btn-default btn-sm active"
                    type="button"
                  >
                    <i
                      className="fa fa-square text-green"
                    />
                  </button>,
                  <button
                    className="btn btn-default btn-sm"
                    type="button"
                  >
                    <i
                      className="fa fa-square text-red"
                    />
                  </button>,
                ],
                "className": "btn-group tools float-right",
                "data-original-title": "Status",
                "data-toggle": "tooltip",
                "title": "",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      className="fa fa-square text-green"
                    />,
                    "className": "btn btn-default btn-sm active",
                    "type": "button",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fa fa-square text-green",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "button",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      className="fa fa-square text-red"
                    />,
                    "className": "btn btn-default btn-sm",
                    "type": "button",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fa fa-square text-red",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "button",
                },
              ],
              "type": "div",
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Conversations />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "data": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "main",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="input-group input-group-sm"
            >
              <input
                className="form-control"
                placeholder="Type message..."
              />
              <button
                className="input-group-addon"
                type="button"
              >
                <i
                  className="fa fa-plus"
                />
              </button>
            </div>,
            "className": "box-border-top",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <input
                  className="form-control"
                  placeholder="Type message..."
                />,
                <button
                  className="input-group-addon"
                  type="button"
                >
                  <i
                    className="fa fa-plus"
                  />
                </button>,
              ],
              "className": "input-group input-group-sm",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "form-control",
                  "placeholder": "Type message...",
                },
                "ref": null,
                "rendered": null,
                "type": "input",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fa fa-plus"
                  />,
                  "className": "input-group-addon",
                  "type": "button",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fa fa-plus",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "button",
              },
            ],
            "type": "div",
          },
          "type": "footer",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
